wolfssl/src/lib.rs

```diff
@@ -202,6 +202,11 @@ impl Protocol {
     fn is_tls_13(&self) -> bool {
         matches!(self, Self::TlsClientV1_3 | Self::TlsServerV1_3)
     }
+
+    /// Checks if the method is compatible with DTLS 1.3
```

Comment:
method -> Protocol

Reply:
It's an odd one this, the WolfSSL naming is `wolfDTLS_*_method` but the type here is `Protocol` (which is a better name for our API IMHO).

I think you are right though and the same thing applies to the existing `is_tls_13` method.